tests/es/climate_HassClimateGetTemperature.yaml

```diff
@@ -3,11 +3,20 @@ tests:
   - sentences:
       - "¿cuál es la temperatura?"
       - "¿qué temperatura hace?"
+      - "¿qué temperatura hay?"
+      - "¿cuántos grados hace?"
+      - "¿cuántos grados hay?"
+      - "¿a cuántos grados estamos?"
     intent:
       name: "HassClimateGetTemperature"
   - sentences:
       - "¿cuál es la temperatura en la cocina?"
       - "¿qué temperatura hace en la cocina?"
+      - "¿qué temperatura hay en la cocina?"
+      - "¿cuántos grados hace en la cocina?"
+      - "¿cuántos grados hay en la cocina?"
+      - "¿a cuántos grados estamos en la cocina?"
+      - "¿a cuántos grados están en la cocina?"
```

Comment:
- "¿A qué temperatura está la cocina?"

Reply:
Added. Thank you.

Reply:
Missing accent in `qué`

Reply:
I told you I am a snob, I wasn't lying 😄

Reply:
Fixed. Thank you. Not a snob, just thorough, which I also try to be 😄 .